knexfile.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 
   test: {
     client: 'pg',
-    connection: 'postgres://localhost/',
+    connection: 'postgres://localhost/indecision',
```

Comment:
I think it makes sense everyone has the same server name, so we don't run into any unnecessary bugs.